docs/self-hosting/configuration/envars.mdx

```diff
@@ -439,6 +439,14 @@ When set, all visits to the Infisical login page will automatically redirect use
   information.
 </Accordion>
 
+## External Database Connections
+
+<ParamField query="DB_SSL_REJECT_UNAUTHORIZED" type="boolean" default="true" optional>
+    Whether external database connections should reject unauthorized SSL certificates. Only set this to `false` if
+    the database you are connecting to is within your internal network.
```

Comment:
we should proabably rephase this to say that we highly-recommend keeping it as "true" for production use-cases

Reply:
minor typos
<img width="1114" alt="image" src="https://github.com/user-attachments/assets/b4c8d941-7eb0-49ea-9691-bad78fb0fb9b" />

same thing in mssql